[DOC] Add each addon documentation in it folder

abdounasser202 opened the pull request:
When we have lot of modules, it's better to document each one on his folder and sumarize them in the repo readme

abdounasser202 commented:
@Yenthe666  @sswapnesh @wardm95 in this pull request, I suggest reordering the documentation

Yenthe666 commented:
@abdounasser202 thanks a lot for your PR, I really appreciate it 😄 I'm a bit torn about this one.
People onlytend to (or I think so) look at the main page of the repository to find into/context about the apps.
What do you think? Maybe an idea is to list the apps with a quick link on the main `README` to have a quick jump to the info page per app that you made? 🤔 
@sswapnesh what is your take?

abdounasser202 commented:
Yep @Yenthe666 this is what is done
In the main page of the repo there are links to the modules that shows a summary for each addon and when you click there you go to the full module explanation

sswapnesh commented:
Hello @abdounasser202  

Initially we had fewer apps to publish here but now it seems we might have few more in future, which will make main README longer (with images too). @Yenthe666  IMO it's good idea to split it module wise.

Yenthe666 commented on README.md:
```diff
@@ -1,54 +1,8 @@
+- [webshop_quick_sell_product_accessory](webshop_quick_sell_product_accessory/README.md): allow to quick add accessories to the basket from the product (in the webshop)
+- [product_alias](product_alias/README.md): allow to set aliases on products so they can be found with multiple names.
+- **webshop_order_customer_reference**: Customers can add their own reference on a webshop order
+- **webshop_quick_sell_product_accessory_public_prices**: Quick Sell Product accessories with public prices
```
These two last apps shouldn't be in bold

sswapnesh replied:
`webshop_order_customer_reference` and `webshop_quick_sell_product_accessory_public_prices` doesn't have and README, maybe we should add with some text/images

Yenthe666 replied:
Would be ideal yes, but either way it does not need the ** for sure.

abdounasser202 replied:
Bold removed as suggested @Yenthe666

Yenthe666 reviewed (changes requested):
Just one note - rest looks good

abdounasser202 committed:
[DOC] Add each addon documentation in it folder

When we have lot of modules, it's better to document each one on his folder and sumarize them in the repo readme

Yenthe666 commented:
Thank you 🙏
